lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -115,26 +133,36 @@ $-separator-color: var(--nx-swatch-indigo-40);
     margin: 0;
   }
 
-  &:focus {
+  &:focus:not(.selected) {
     border-radius: 0;
-    outline: 2px solid $-focus-color;
-    outline-offset: -2px;
+    outline: var(--nx-focus-outline-width) solid var(--nx-color-global-sidebar-menu-icon-border-focus);
+    outline-offset: calc(0px - var(--nx-focus-outline-width));
+  }
+
+  &:focus, &:focus:hover {
+    border-radius: 0;
+    outline: var(--nx-focus-outline-width) solid var(--nx-color-global-sidebar-link-border-focus);
+    outline-offset: calc(0px - var(--nx-focus-outline-offset-global-sidebar-link) - var(--nx-focus-outline-width));
   }
 
   &:hover {
-    background-color: $-hover-color;
+    @include link-no-outline;
+    background-color: var(--nx-color-global-sidebar-link-hover);
   }
 
   &:active {
-    background-color: $-active-color;
+    @include link-no-outline;
+    background-color: var(--nx-color-global-sidebar-link-active);
   }
 
   &.selected {
-    background-color: var(--nx-swatch-blue-40);
+    @include link-no-outline;
+    background-color: var(--nx-color-global-sidebar-link-selected);
   }
 
   &:active:focus {
-    color: $-text-color;
+    color: var(--nx-color-global-sidebar-link-text-interactive);
```

Comment:
This seems redundant with the color being set by the mixin in the three blocks above.  I do see however that the default styles for `.nx-text-link` are interfering due to them having equal specificity.  A better way to solve that would be to select `.nx-global-sidebar__navigation.nx-text-link:...`

Reply:
> This seems redundant with the color being set by the mixin in the three blocks above. I do see however that the default styles for `.nx-text-link` are interfering due to them having equal specificity. A better way to solve that would be to select `.nx-global-sidebar__navigation.nx-text-link:...`

See https://github.com/sonatype/sonatype-react-shared-components/pull/1091#discussion_r1180646547